openssh/Patches/0008-Add-DisableBanner-option.patch

```diff
@@ -135,15 +135,15 @@ index 1563aae3..2bcc365d 100644
  Specifies that a TCP port on the local machine be forwarded
  over the secure channel, and the application
 diff --git a/sshconnect2.c b/sshconnect2.c
-index a53ab95d..65f8e1c9 100644
+index a53ab95d..8573130e 100644
 --- a/sshconnect2.c
 +++ b/sshconnect2.c
 @@ -85,6 +85,10 @@ extern char *client_version_string;
  extern char *server_version_string;
  extern Options options;
  
 +#ifdef DISABLE_BANNER
-+extern struct sshbuf command;
++extern struct sshbuf *command;
```

Comment:
This is the crux of our mistake, right?

Reply:
Yes, though the bit below that is necessary as well (checking whether this pointer is NULL and otherwise checking the length of the buffer)

Reply:
Oh, definitely. I just wanted to make sure that this was the start of where things needed to get fixed.

Also, see my other outside the code about the diff noise here. If it's smoother for you to have all of this to land, so be it.  I was hoping to qualify the OpenSSL 3 branch (which has been around for a while and getting hammered/tested) for landing this week.  (Less merge hell, the better.)

But to prevent merge hassle I'd love if this was just a change to this 0008 patch. If you think that's unreasonable, I'll take this as-is.